pkgs/development/libraries/mpich/default.nix

```diff
@@ -45,6 +60,10 @@ stdenv.mkDerivation  rec {
   '';
 
   meta = with lib; {
+    # --with-pmix silently disables gforker,
+    # cf. the comment about pmixSupport above
+    broken = pmixSupport && processManagers != ["slurm"];
```

Comment:
Note that "slurm" is not a valid process manager here. The line `mpich.override { pmixSupport = true; withPm = [ "slurm" ]; };` only works because mpich ignores the `--with-pm` option once you turn on PMIx.

Reply:
Oh wow, I thought pmix is the principal thing to support slurm 😅 What's there not to love about autotools